src/base-input.js

```diff
@@ -131,14 +131,25 @@ export default class BaseInput extends LitElement {
    * @returns {string} Replaces innerHTML with empty string.
    */
   handleClickClear() {
+    let click = null;
+
     this.inputElement.value = "";
     this.value = "";
     this.classList.remove("passwordIcon--show");
     this.focus();
 
+    click = new CustomEvent('clearElement', {
```

Comment:
I think it should be possible to dispatch an input event instead

```js
new Event('input', {
  bubbles: true,
  composed: true
}
```

This way, users of the component don't have to set up two event listeners. Either way, we should make sure to test IE (I can help with that) -- it doesn't support CustomEvent or Event constructors. We may need a guard statement.

Reply:
A guard statement from what? Not following you?

Reply:
Custom event or creating a new input event ¯\_(ツ)_/¯ 

I don't really have a spec with a defined done-when. https://github.com/AlaskaAirlines/auro-input/issues/2#issuecomment-712256637

Because of the vagueness of this, it could go either way. @liambrozik ... what is your input on this?

Reply:
IE doesn't support the Event constructor, so we might need to guard against it before initializing an event. I'm not sure, we'll need to test.
```js
if ( typeof window.Event !== "function" ) return;
```

But my main suggestion is to dispatch an `input` event instead of a custom `clearElement` event.

Reply:
I would agree with @geoffrich here, I already have event listeners for `input` events and making the clear action an `input` event as well would make sense.

Reply:
Fair enough. Updated.